src/components/Data/Layout.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <Multipane class="DataLayout Custom-resizer" layout="vertical">
-    <div class="DataLayout-sidebarWrapper" data-cy="DataLayout-sidebarWrapper">
+  <Multipane class="DataLayout Custom-resizer" layout="vertical" @paneResizeStop="registerNewPaneSize($event)">
+    <div class="DataLayout-sidebarWrapper" :style="{ width: this.paneSize}" data-cy="DataLayout-sidebarWrapper">
```

Comment:
Code formatting (if you don't do it here, it will end up in someone else's PR).
```suggestion
    <div class="DataLayout-sidebarWrapper" :style="{ width: this.paneSize }" data-cy="DataLayout-sidebarWrapper">
```